devices/baseEstimatorV1/include/baseEstimatorV1.h

```diff
@@ -547,7 +547,7 @@ namespace yarp {
 
         const double m_sensor_timeout_in_seconds{2.0}; ///< Timeout to check for sensor measurements during dry run initial check
         const size_t m_nr_of_channels_in_YARP_IMU_sensor{12}; ///< Number of channels available in YARP IMU sensor output port
-        const size_t m_nr_of_channels_in_YARP_FT_sensor{6}; ///< Number of channels available in YARP FT sensor output port
+        const size_t m_nr_of_channels_in_YARP_ft{6}; ///< Number of channels available in YARP FT sensor output port
```

Comment:
here you updated the name of a method in the source code.. I'm not sure it's relevant.. But I guess it's harmless if you update all it's occurrences in the code..

Reply:
No, you are right. That variable should not be changed.